docs/sources/tutorials/get-started.md

```diff
@@ -71,17 +87,169 @@ services:
           editable: false
         EOF
         /run.sh
-    image: grafana/grafana:latest
+    image: grafana/grafana:11.0.0
     ports:
       - "3000:3000"
 ```
 
-After running `docker-compose up`, open [http://localhost:3000](http://localhost:3000) in your browser to view the Grafana UI.
+Run `docker compose up` to start your Docker container and open [http://localhost:3000](http://localhost:3000) in your browser to view the Grafana UI.
+
+ {{< admonition type="note" >}}
+If you the following error when you start your Docker container, `docker: 'compose' is not a docker command`, use the command `docker-compose up` to start your Docker container.
+ {{< /admonition >}}
+
+## Configure {{< param "PRODUCT_NAME" >}}
+
+Once the local Grafana instance is set up, the next step is to configure {{< param "PRODUCT_NAME" >}}.
+You use components in the `config.alloy` file to tell {{< param "PRODUCT_NAME" >}} which logs you want to scrape, how you want to process that data, and where you want the data sent.
+
+The examples run on a single host so that you can run them on your laptop or in a Virtual Machine.
+You can try the examples using a `config.alloy` file and experiment with the examples yourself.
+
+For the following steps, create a file called `config.alloy` in your current working directory. 
+If you have enabled the {{< param "PRODUCT_NAME" >}} UI, you can "hot reload" a configuration from a file.
+In a later step, you will copy this file to where {{< param "PRODUCT_NAME" >}} will pick it up, and be able to reload without restarting the system service.
+
+### First Component: Log files
+
+Paste this component into the top of the `config.alloy` file:
+
+```alloy
+local.file_match "local_files" {
+    path_targets = [{"__path__" = "/var/log/*.log"}]
+    sync_period = "5s"
+}
+```
+
+This component creates a [local.file_match][] component named `local_files` with an attribute that tells {{< param "PRODUCT_NAME" >}} which files to source, and to check every 5 seconds.
+
+### Second Component: Scraping
+
+Paste this component next in the `config.alloy` file:
+
+```alloy
+loki.source.file "log_scrape" {
+    targets    = local.file_match.local_files.targets
+    forward_to = [loki.write.grafana_loki.receiver]
+    tail_from_end = true
+}
+```
+
+This configuration creates a [loki.source.file][] component named `log_scrape`, and shows the pipeline concept of {{< param "PRODUCT_NAME" >}} in action. The `log.scrape` component does the following:
+
+1. It applies to the `local_files` component (its "source" or target).
```

Comment:
The usage of applies here is throwing me off a bit, as I'm reading "it applies" as an action performed by the log_scrape component. What does it mean for log_scrape to apply to local_files?

Reply:
not sure what to do to clarify wording here.  The reason we're taking it slow is to explain interconnectivity of the config component.  Here, "applies to" means that because local_files is the source, this component applies to that input.

Reply:
Given we're explaining interconnectivity, maybe "connects?" 

> It connects to the `local_files` component (its "source" or target).